.travis.yml

```diff
@@ -27,4 +28,4 @@ install:
   - bundle _1.12.4_ install --jobs=3 --retry=3 --path=${BUNDLE_PATH:-vendor/bundle}
 
 before_script:
-  - 'cp config/database.yml.travis config/database.yml'
+  - psql -c 'create database travis_ci_test;' -U postgres
```

Comment:
We should keep the config so the tests are as close to what people run in dev/prod as possible and if a name changes this doesn't get lost. Also the tests use the database.yml to make the connection.